templates/ssr/customServer.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine AS builder
```

Comment:
We also discussed that there will be 3 options now 

1. static export
2. SSR default 
3. SSR custom